src/WebUI/Controllers/DocumentHistoryGridController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DKP.InvestmentReview.WebUI.Controllers
+{
+    public class DocumentHistoryGridServiceController : ApiController
+    {        
+        [HttpGet]
+        public List<HistoryGridData> getDocumentHistoryGridDetails()
+        {
+            List<HistoryGridData> objGridData = new List<HistoryGridData>();
+            objGridData.Add(new HistoryGridData("Benchmark Return", "Template 1 - Instance 1", "01/01/2020"));
+            objGridData.Add(new HistoryGridData("Annual Report: ABC", "Template 1 - Instance 1", "01/01/2020"));
+            objGridData.Add(new HistoryGridData("Annual Report: XYZ", "Template 1 - Instance 1", "01/01/2020"));
+            objGridData.Add(new HistoryGridData("2020 Sales History", "Template 1 - Instance 1", "01/01/2020"));
+
+            return objGridData;
+        }
+    }
+
+    public class HistoryGridData
```

Comment:
💡 Generally we should return mock data from application layer, see `Application/Components/BenchmarkReturns\Mock`

Reply:
This is in someone else's code. Came in this PR because I merged his code. But this is noted and will make sure this is fixed later when actually working on that component.